app/src/main/java/org/grapheneos/apps/client/service/SeamlessUpdaterJob.kt

```diff
@@ -74,7 +74,7 @@ class SeamlessUpdaterJob : JobService() {
 
                 if (requireConfirmation.isNotBlank() && result.requireConfirmation.isNotEmpty()) {
                     if (content.isNotBlank()) content += ", "
-                    content += "$requireConfirmation update update available."
```

Comment:
Saying that an update is available may be a better way to phrase it. A problem with changing so much at once is that the whole pull request will be stalled or rejected based on any single minor thing that's not desired.